core/azure-core-amqp/README.md

```diff
@@ -0,0 +1,50 @@
+# Azure Core AMQP client library for Java
+
+Azure Core AMQP client library is a collection of classes common to the AMQP protocol. It help developers create their 
+own AMQP client library that abstracts from the underlying transport library's implementation.
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+
+### Adding the package to your product
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-core-amqp</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+</dependency>
+```
+
+## Key concepts
+
+The concepts for AMQP are well documented in 
+[OASIS Advanced Messaging Queuing Protocol (AMQP) Version 1.0 ](http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-overview-v1.0-os.html).
+
+## Examples
+
+TODO: Fill something out.
+
+## Troubleshooting
+
+TODO: Fill something out.
+
+## Next steps
+
+TODO: Fill something out. Maybe read the docs?
+
+## Contributing
+
+If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.io/guidelines.html).
+
+1. Fork it
+1. Create your feature branch (`git checkout -b my-new-feature`)
+1. Commit your changes (`git commit -am 'Add some feature'`)
+1. Push to the branch (`git push origin my-new-feature`)
+1. Create new Pull Request
+
+<!-- Links -->
+[jdk]: https://docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable
```

Comment:
This link 404's. I would just remove the link entirely.

Reply:
I copied it from the application configuration README. Lol. Should I remove that too?
How about this link? 🤔 
https://docs.microsoft.com/en-us/java/azure/java-sdk-azure-get-started?view=azure-java-stable

Reply:
I would sooner just remove the link from all places. Leave it as an exercise to the reader to find the JDK they want to use.

Reply:
Gotcha. :) Sounds good. Removed.